doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -304,8 +304,8 @@ Hints used by :tacn:`auto`, :tacn:`eauto` and other tactics are stored in hint
 databases.  Each database maps head symbols to a list of hints.  Use the
 :cmd:`Print Hint` command to view a database.
 
-Each hint has a cost that is a nonnegative
-integer and an optional pattern. Hints with lower costs are tried first.
+Each hint has a priority number and an optional pattern. Hints with lower
+priority numbers are tried first.
```

Comment:
The term "cost" is used more than a dozen times on this page, not sure changing it only in that sentence is the best solution.